README.md

```diff
@@ -154,7 +183,7 @@ How many valid sentences are there in this language?
 
 ## A2.6
 
-«replace this with your answer»
+60  
```

Comment:
# 0

the grammar is infinite, as you can have an arbitrary number of adjectives.